core/src/main/java/com/novoda/noplayer/internal/exoplayer/CompositeTrackSelectorCreator.java

```diff
@@ -25,8 +26,10 @@ CompositeTrackSelector create(Options options, DefaultBandwidthMeter bandwidthMe
                 Clock.DEFAULT
         );
         DefaultTrackSelector trackSelector = new DefaultTrackSelector(adaptiveTrackSelectionFactory);
+        Dimension maxVideoSize = options.maxVideoSize();
         DefaultTrackSelector.Parameters trackSelectorParameters = trackSelector.buildUponParameters()
                 .setMaxVideoBitrate(options.maxVideoBitrate())
+                .setMaxVideoSize(maxVideoSize.width(), maxVideoSize.height())
```

Comment:
What is the behavior now when we set both max bitrate and max size? Should we add a description of the behavior in the API documentation?

Reply:
Mmm both methods have their own Javadoc. The behaviour will be that both limits will be applied.
- No track will be selected if the bitrate of the track is higher than the set in `setMaxVideoBitrate`
- No track will be selected if the video size is higher than the set in `setMaxVideoSize`